pkg/cmd/roachprod-microbench/compare.go

```diff
@@ -318,6 +355,171 @@ func (c *compare) compareUsingThreshold(comparisonResultsMap model.ComparisonRes
 	return nil
 }
 
+func (c *compare) createBenchSeries() ([]*benchseries.ComparisonSeries, error) {
+	opts := benchseries.DefaultBuilderOptions()
+	opts.Experiment = "run-time"
+	opts.Compare = "cockroach"
+	builder, err := benchseries.NewBuilder(opts)
+	if err != nil {
+		return nil, err
+	}
+
+	var benchBuf bytes.Buffer
+	readFileFn := func(filePath string, required bool) error {
+		data, err := os.ReadFile(filePath)
+		if err != nil {
+			if !required && oserror.IsNotExist(err) {
+				return nil
+			}
+			return errors.Wrapf(err, "failed to read file %s", filePath)
+		}
+		benchBuf.Write(data)
+		benchBuf.WriteString("\n")
+		return nil
+	}
+
+	for k, v := range c.influxConfig.metadata {
+		benchBuf.WriteString(fmt.Sprintf("%s: %s\n", k, v))
+	}
+
+	logPaths := map[string]string{
+		"experiment": c.experimentDir,
+		"baseline":   c.baselineDir,
+	}
+	for logType, dir := range logPaths {
```

Comment:
I assume this is true, but to confirm: the order in which we write the data to the buffer does not change anything observable from outside this function, right?

Reply:
Yes, order should not matter here. The metadata prefixing the data within the loop is the most important aspect.